README.md

```diff
@@ -15,7 +15,11 @@ Step-up Gateway
 
 Clone the repository or download the archive to a directory. Install the dependencies by running `composer install` and fill out the database credentials et cetera.
 
+Make sure to run database migrations for u2f using `app/console u2f:migrations:migrate`.
+
+Run `app/console mopa:bootstrap:symlink:less` to configure Bootstrap symlinks.
```

Comment:
I executed `app/console mopa:bootstrap:install:font`, but I'm not sure that's actually required.